tests/test_pyrenew_hew_data.py

```diff
@@ -123,15 +123,45 @@ def test_to_forecast_data(
     assert (
         (forecast_data.first_hospital_admissions_date - data.first_data_date_overall)
         / np.timedelta64(1, "D")
-    ).item() <= 6
+    ).item() <= 7
```

Comment:
There are no changes to the `parametrize` setup, so why does this need to change?

Reply:
>But hospital admissions are weekly data - you need at least one full week!

By this logic, we should always be jumping to the second Saturday and requiring `7 <= forecast_data.first_hospital_admissions_date - data.first_data_date_overall < 14 `, no?